sky/serve/autoscalers.py

```diff
@@ -66,9 +66,22 @@ def __init__(self, spec: 'service_spec.SkyServiceSpec') -> None:
             target_num_replicas: Target number of replicas output by autoscaler.
         """
         self.min_replicas: int = spec.min_replicas
-        self.max_replicas: int = spec.max_replicas or spec.min_replicas
+        self.max_replicas: int = (spec.max_replicas if spec.max_replicas
+                                  is not None else spec.min_replicas)
+        # Target number of replicas is initialized to min replicas.
+        # TODO(MaoZiming): add init replica numbers in SkyServe spec.
         self.target_num_replicas: int = spec.min_replicas
 
+    def update_version(self, version: int,
+                       spec: 'service_spec.SkyServiceSpec') -> None:
+        del version  # Unused.
```

Comment:
Should we update the version in the base Autoscaler class instead? It seems to me that the version should be required for all inheritance.

Reply:
Added version updating to the base class.